go-base/Dockerfile

```diff
@@ -2,17 +2,36 @@ FROM ubuntu:16.04
 
 RUN apt-get update 
 RUN apt-get install -y build-essential git
+
+# rumprun can only be built with gcc-6
+# XXX: Not only that, this version has to match whatever
+# gcc was used to build rumprun in the host.
+RUN apt-get update && \
+	apt-get install build-essential software-properties-common -y && \
+	add-apt-repository ppa:ubuntu-toolchain-r/test -y && \
+	apt-get update && \
+	apt-get install gcc-snapshot -y && \
+	apt-get update && \
+	apt-get install gcc-6 g++-6 -y && \
+	update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 60 --slave /usr/bin/g++ g++ /usr/bin/g++-6 && \
+	apt-get install gcc-4.8 g++-4.8 -y && \
+	update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 60 --slave /usr/bin/g++ g++ /usr/bin/g++-4.8
+
 RUN mkdir /gopath
 RUN mkdir /goapp
 
-ADD rumprun-solo5 /root/unikernels/nabla-base-build/rumprun
+# XXX: this is pretty inconvenient. Turns out that rumprun
+# needs to be at exactly the same place where it was built.
+# (unless I'm missing something).
+ARG host_rumproot
+ADD rumprun-solo5 ${host_rumproot}
```

Comment:
Dockerfile best practices: Use `COPY` instead of `ADD`. This is because `ADD` is overloaded.

Reply:
done